console.log(soma(1, 1));
// Por ser chamada antes de sua criação, vai dar erro, caso fosse uma função "normal", ela funcionaria.
const soma = function(num1, num2){return num1 + num2}

// console.log(soma(1, 1));